app/views/admissions_admin/applicants/show_unlogged_applicants.html.haml

```diff
@@ -0,0 +1,36 @@
+= stylesheet_link_tag("admissions/unlogged_applicants")
+
+%h2= t('admissions_admin.unlogged_applicants')
+
+- unless @applicants.empty?
```

Comment:
Here we check that `@applicants` is empty, but below we iterate over `@group_jobs` without checking if it's empty, but if the first is empty, then also the second should be empty. This should be improved, though.